backend/packages/Upgrade/src/api/services/ExperimentAssignmentService.ts

```diff
@@ -513,9 +515,9 @@ export class ExperimentAssignmentService {
     let userDoc = await this.experimentUserService.getOriginalUserDoc(userId);
     const keyUniqueArray = [];
 
-    // create user if user does not exist
+    // throw error if user not defined
     if (!userDoc) {
-      userDoc = await this.userRepository.save({ id: userId });
+      throw new Error(`User not defined: ${userId}`);
```

Comment:
@YashilDepani  Shouldn't we throw the same error for all apis? In the `getAllExperimentConditions()`, we are throwing with `type as EXPERIMENT_USER_NOT_DEFINED`.

Reply:
The same type of error is also thrown by this /bloblog. Reference code: 
`throw new Error(
        JSON.stringify({
          type: SERVER_ERROR.EXPERIMENT_USER_NOT_DEFINED,
          message: error.message,
        })
      );`

Reply:
@YashilDepani Please update the code here to send the same error which we are throwing in other client APIs